pkg/util/mount/mount.go

```diff
@@ -56,9 +56,11 @@ type SafeFormatAndMount struct {
 	Runner exec.Interface
 }
 
-// Mount mounts the given disk. If the disk is not formatted and the disk is not being mounted as read only
-// it will format the disk first then mount it.
-func (mounter *SafeFormatAndMount) Mount(source string, target string, fstype string, options []string) error {
+// FormatAndMount mounts the given disk. If the disk is not formatted
+// and it is not being mounted as read-only it will format the disk
+// first then mount it. If the disk is already formatted or it is
+// being mounted as read-only it will be mounted without formatting.
+func (mounter *SafeFormatAndMount) FormatAndMount(source string, target string, fstype string, options []string) error {
```

Comment:
Can you put more words on usage? Distinguishing `Mount` and `SafeFormatAndMount` doesn't look quite easy to digest on the first glance.